gmso/external/convert_foyer_xml.py

```diff
@@ -357,7 +363,7 @@ def _write_ub_angles(forcefield, ff_kwargs):
         forcefield,
         "AngleTypes",
         attrib_dict={
-            "expression": "k * (w - w_0) ** 2",
+            "expression": "0.5 * k * (w - w_0) ** 2",
```

Comment:
We should be consistent with the above or we go with 0.5 everywhere
```suggestion
            "expression": "1/2 * k * (w - w_0) ** 2",
```